src/components/SystemHealthIcon/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import { IconStatus } from 'types/iconStatus';
+
+type SystemHealthIconProps = {
+  status: IconStatus;
+  size: 'medium' | 'xl';
+  label?: string;
+};
+
+const SystemHealthIcon = ({ status, size, label }: SystemHealthIconProps) => {
+  const classes = classnames(
+    'fa',
+    {
+      'system-health-icon-success': status === 'success',
```

Comment:
Should these classnames be moved to a .scss file in /SystemHealthIcon, rather than in /SystemProfileHealthCard?

Reply:
Exactly!